bankaccount.go

```diff
@@ -24,7 +24,7 @@ type BankAccountParams struct {
 
 	// Information on an external account to reference. Only used if `Token`
 	// is not provided.
-	Account, Country, Currency, Routing string
+	Account, Country, Currency, Routing, AccountHolderType, Name string
```

Comment:
Nit, but mind keeping this list in alphabetical order since it's that way already Thanks!